src/workloads/query/FilterWithComplexLogicalExpression.yml

```diff
@@ -25,6 +25,8 @@ Keywords:
 - QuiesceActor
 - insert
 - find
+- matcher
```

Comment:
I think these were added to the wrong file?

Reply:
yeah wrong file unless you meant to add to both.

Reply:
Oops, fixed now.